[2024-04-30T16:53:48.757+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2024-04-30T16:53:43.073401+00:00 [queued]>
[2024-04-30T16:53:48.807+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2024-04-30T16:53:43.073401+00:00 [queued]>
[2024-04-30T16:53:48.810+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-04-30T16:53:48.811+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2024-04-30T16:53:48.812+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-04-30T16:53:48.893+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_available> on 2024-04-30 16:53:43.073401+00:00
[2024-04-30T16:53:48.916+0000] {standard_task_runner.py:55} INFO - Started process 613 to run task
[2024-04-30T16:53:48.952+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'is_api_available', 'manual__2024-04-30T16:53:43.073401+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpdhf8i28d']
[2024-04-30T16:53:48.980+0000] {standard_task_runner.py:83} INFO - Job 8: Subtask is_api_available
[2024-04-30T16:53:49.184+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.is_api_available manual__2024-04-30T16:53:43.073401+00:00 [running]> on host ec0cac6d8211
[2024-04-30T16:53:49.272+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2024-04-30T16:53:43.073401+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-04-30T16:53:43.073401+00:00
[2024-04-30T16:53:49.274+0000] {http.py:120} INFO - Poking: api/
[2024-04-30T16:53:49.289+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2024-04-30T16:53:49.293+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2024-04-30T16:53:49.600+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2024-04-30T16:53:49.618+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=is_api_available, execution_date=20240430T165343, start_date=20240430T165348, end_date=20240430T165349
[2024-04-30T16:53:49.681+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2024-04-30T16:53:49.729+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
